jme3-core/src/main/java/com/jme3/animation/SkeletonControl.java

```diff
@@ -443,9 +459,19 @@ public Node getAttachmentsNode(String boneName) {
                     + "in the skeleton.");
         }
 
-        Node n = b.getAttachmentsNode();
-        Node model = (Node) spatial;
-        model.attachChild(n);
+        updateTargetsAndMaterials(spatial);
+        Node n = b.getAttachmentsNode(targetGeometry);
```

Comment:
If I pass a list of geometries to the bone, and the bone affects more than one of them, will it be acceptable for the bone to arbitrarily use the first one it affects?